radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -0,0 +1,284 @@
+/**
+ * @file RAntenna.c
+ * @date Jan 15, 2022
+ * @author Addi Amaya (caa746)
+ */
+
+#include <RAntenna.h>
+#include <satellite-subsystems/IsisAntS.h>
+#include <string.h>
+#include <hal/errors.h>
+#include <hal/Timing/Time.h>
+
+/***************************************************************************************************
+                                  PRIVATE DEFINITIONS AND VARIABLES
+***************************************************************************************************/
+
+/** Struct for defining Antenna Systems I2C Addresses. Configured as a single bus, both address write to same location */
+ISISantsI2Caddress RAntennaI2Caddress = {ANTENNA_I2C_SLAVE_ADDR_PRIMARY,ANTENNA_I2C_SLAVE_ADDR_REDUNANT};
+
+/** Struct that holds the current deployment status of the antenna*/
+antenna_deployment_status_t RAntennaStatus = { 0 };
```

Comment:
Could/should these be made private with a `static` keyword? Or are they used elsewhere? Also technically the leading R is not necessary based on our coding standard but I'm not too worried about that.